### PR TITLE
Release notes for 0.25.0

### PR DESCRIPTION
**Pull request by hsaraogi**
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/1264)

### CODE REVIEW

**Comment by SerialVelocity** [comment]
Can you create a new develop block too please?

File: docs/source/release_notes/release-notes.rst [hunk 1]
```diff
@@ -42,34 +42,39 @@ develop
          - Change
 
     *    - |new|
+         - ConnectionSupplier consumers can now choose to receive a brand new unshared connection.
```
Comment (gsheasby) [inline docs/source/release_notes/release-notes.rst]: On its own, this doesn't mean much to me. I suggest prefixing it with `OracleKVS:` (or DbKvs if it works for both that and postgres).

File: docs/source/release_notes/release-notes.rst [hunk 2]
```diff
@@ -42,34 +42,39 @@ develop
...
     *    - |fixed|
          - Oracle will not drop a table that already exists on  ``createTable`` calls when multiple AtlasDB clients make the call to create the same table.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1243>`__)
 
+    *    - |fixed|
+         - Certain Oracle KVS calls no longer attempt to leak connections created internally.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1215>`__)
+
     *    - |breaking|
          - Our jackson version has been updated from 2.5.1 to 2.6.7 and dropwizard version from 0.8.2 to 0.9.3.
```
Comment (gsheasby) [inline docs/source/release_notes/release-notes.rst]: nit: Jackson is capitalised.

File: docs/source/release_notes/release-notes.rst [hunk 3]
```diff
@@ -31,7 +31,7 @@ Changelog
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 =======
-develop
+v0.25.0
```
Comment (gsheasby) [inline docs/source/release_notes/release-notes.rst]: I think the general practice is to add `v0.25.0` as a new row, and leave the `develop` one intact (but with an empty list-table).

**Pull request merged**